Vehicle_Register_Schema.json

```diff
@@ -0,0 +1,183 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "type": "array",
+  "items": {
+    "$ref": "#/definitions/WelcomeElement"
+  },
+  "definitions": {
+    "WelcomeElement": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "vehicle_type": {
+          "$ref": "#/definitions/VehicleType"
+        },
+        "public_vehicle_number": {
+          "type": "string",
+          "format": "integer"
+        },
+        "status": {
+          "$ref": "#/definitions/Status"
+        },
+        "vehicle_fuel_source": {
+          "$ref": "#/definitions/VehicleFuelSource"
+        },
+        "wheelchair_accessible": {
+          "$ref": "#/definitions/WheelchairAccessible"
+        },
+        "company_name": {
+          "type": "string"
+        },
+        "address": {
+          "type": "string"
+        },
+        "city": {
+          "$ref": "#/definitions/City"
+        },
+        "state": {
+          "$ref": "#/definitions/State"
+        },
+        "zip_code": {
+          "type": "string",
+          "format": "integer"
+        },
+        "record_id": {
+          "type": "string"
+        },
+        "vehicle_make": {
+          "type": "string"
+        },
+        "vehicle_model": {
+          "type": "string"
+        },
+        "vehicle_model_year": {
```

Comment:
Ditto note above... might as well add some data validation while you're in here.